recipes/asimov/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "asimov" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 160df0c977cf5c65794f244957c7f4436d250bbf63fd944654219bf29b9114e3
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - locutus=asimov.locutus:cli
+    - olivaw=asimov.olivaw:olivaw
+    - asimov=asimov.olivaw:olivaw
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools
+    - setuptools_scm
+  run:
+    - click
+    - gitpython
+    - gwpy
+    - ligo-gracedb
+    - liquidpy
+    - lscsoft-glue
+    - igwn-auth-utils
+    - networkx
+    - numpy
+    - otter-report >=0.3.3
+    - python >=3.7
+    - python-gitlab
+    - python-htcondor
+    - pytz
+    - pyyaml
+    - requests
+    - flask
+    - tinydb
+
+test:
+  requires:
+    - git
```

Comment:
Why is `git` required here?

Reply:
It is required for the tests, but I can remove it if you think this is a problem?

Reply:
I've removed it as it's not needed if gitpython has been installed already.